tests/core/data/test_excel_data_node.py

```diff
@@ -263,6 +259,38 @@ def test_write(self, excel_file, default_data_frame, content, columns):
         excel_dn.write(None)
         assert len(excel_dn.read()) == 0
 
+    @pytest.mark.parametrize(
+        "content,columns,sheet_name",
+        [
+            ([{"a": 11, "b": 22, "c": 33}, {"a": 44, "b": 55, "c": 66}], None, "sheet_name"),
+        ],
+    )
+    def test_write_with_sheet_name(self, excel_file_with_sheet_name, default_data_frame, content, columns, sheet_name):
+        excel_dn = ExcelDataNode(
+            "foo", Scope.SCENARIO, properties={"path": excel_file_with_sheet_name, "sheet_name": sheet_name}
+        )
+        excel_dn.read()
+        assert np.array_equal(excel_dn.read().values, default_data_frame.values)
+        if not columns:
+            excel_dn.write(content)
+            df = pd.DataFrame(content)
+        else:
+            excel_dn.write_with_column_names(content, columns)
+            df = pd.DataFrame(content, columns=columns)
```

Comment:
In this test, the column is always None. So the `else` will not be called at all, right?

Should we add another test with defined `columns` in the `@pytest.mark.parametrize`?